src/SFA.DAS.FAA.Domain/Entities/SavedSearchEntity.cs

```diff
@@ -21,6 +22,7 @@ public static implicit operator SavedSearchEntity(SavedSearch source)
                 DateCreated = source.DateCreated,
                 LastRunDate = source.LastRunDate,
                 EmailLastSendDate = source.EmailLastSendDate,
+                
```

Comment:
Did you mean to map the token here?

Reply:
yes, UnusbscibeToken will be added in here